### ISSUE TITLE
increase dense nodes

### ISSUE THREAD

**Issue by cwmeijer** [NLeSC/mcfly#47]
I think the current number is way too low.

**Comment by dafnevk** [comment]
- [ ] research literature for what a reasonable number is
- [ ] change default number in code
- [ ] compare model with  many nodes and model with fewer nodes

**Comment by dafnevk** [comment]
Summary of what I found in literature:
**Images**
Some ConvNets have no (additional) dense layers (GoogLeNet, ResNet). Others (AlexNet, VGGnet) use 4096 nodes (after using Conv layers of 264 or 512 filters).

**time series**
* Ordonez 2016 (on which we based the DeepConvLstm model) uses in the CNN models to which they compare: 2 dense layers of 128 nodes (after conv layers of size 64)
* Plotz 2016 uses no hidden dense layers after the conv layers (conv layers are size 16 to 128)
* Zheng 2014 uses one hidden layer with 732 nodes

Note that layer sizes are almost always a power of 2 - I guess this has to do with computational advantages. 
But I wonder if that means we should also make this hyperparameter log-uniform distriubuted (uniformly pick a power of 2). This could reduce the hyperparameter space a lot.

**Comment by dafnevk** [comment]
I created 10 models with 10-50 hidden nodes and 10 models with 500-2000 hidden nodes and trained them on a sample of 500 of the Pamap2 dataset, for 5 epochs.
I did the same thing but for 10 epochs (these were different models)
In 5 epochs, the models with few nodes seem to perform better in general. In 10 epochs, the models with many nodes are better in general. This makes sense, because the models have a larger complexity so they need more iterations to train.
Boxplot of the validation accuracies of the models:
![image](https://cloud.githubusercontent.com/assets/4925047/17555846/5c151986-5f11-11e6-92a8-03436ea90c42.png)
![image](https://cloud.githubusercontent.com/assets/4925047/17556519/fd8cbbbe-5f13-11e6-83bd-f6ad59fd4d1b.png)
